Operators/Types/lib/io/osc/OscOutput.cs

```diff
@@ -0,0 +1,142 @@
+using T3.Core.Operator;
+using T3.Core.Operator.Attributes;
+using T3.Core.Operator.Slots;
+using T3.Core.Logging;
+using System.Net;
+using Rug.Osc;
+using System;
+using T3.Core.Utils;
+
+namespace T3.Operators.Types.Id_4e99da86_482f_4037_8664_b2371526d632
+{
+    public class OscOutput : Instance<OscOutput>
+    {
+        private bool _connected;
+        private IPAddress _address;
+        private int _port;
+        private float _num_value;
+        private string _str_value;
+        private OscSender _sender;
+        public OscOutput()
+        {
+            Result.UpdateAction = Update;
+        }
+
+        private enum OscType
+        {
+            Number,
+            String
+        }
+        private void Update(EvaluationContext context)
+        {
+            var location = Location.GetValue(context);
+            var num = Number.GetValue(context);
+            var str = String.GetValue(context);
+            var oscType = OscValueType.GetEnumValue<OscType>(context);
+
+            CheckAddr(context);
```

Comment:
```suggestion
            if(!TryGetValidAdress(IdAddress.GetValue(context), out var problem, out var address) {
               _lastErrorMessage = problem;
            }
```


I would have expected a `CheckXYZ` method to return a boolean. 
I much prefer the `TryGetXYZ` pattern being as static as possible.